core/payment-driver/base/src/bus.rs

```diff
@@ -145,6 +147,8 @@ pub async fn sign(node_id: NodeId, payload: Vec<u8>) -> Result<Vec<u8>, GenericE
     Ok(signature)
 }
 
+/// Notifies the Payment service that the scheduled payment is processed successfully.
```

Comment:
maybe worth adding "calling this will call `verify_payment` in the provider driver